.github/workflows/e2e-tests-ondemand.yaml

```diff
@@ -6,16 +6,49 @@ on:
       target:
         description: 'Target branch to run E2E tests over'
         required: true
-        # TODO: @andriisoldatenko revisit it
         default: 'release-1.1'
 
 permissions:
   checks: write
 
+# based on:
+# https://docs.dynatrace.com/docs/setup-and-configuration/technology-support/support-model-for-kubernetes
+# https://kubernetes.io/releases/
 jobs:
-  run-in-k8s-1-21:
-    name: Run in Kubernetes 1.21
+  # TODO: End of support: Aug 1, 2024, remove after this date
+  run-k8s-matrix:
+    name: Run in Kubernetes
+    strategy:
+      matrix:
+        flc_env: [ dto-k8s-1-24, dto-k8s-1-25, dto-k8s-1-26, dto-k8s-1-27, dto-k8s-1-28, dto-k8s-1-29 ]
     environment: E2E
+    runs-on:
+      - self-hosted
+      - operator-e2e
+    steps:
+      - name: Checkout
+        uses: actions/checkout@a5ac7e51b41094c92402da3b24376905380afc29 # v4.1.6
+      - name: Run e2e test
+        uses: ./.github/actions/run-e2e
+        with:
+          flc-namespace: dto-k8s-ondemand
+          flc-environment: ${{ matrix.flc_env }}
+          target-branch: ${{ github.event.inputs.target }}
+          tenant1-name: ${{ secrets.TENANT1_NAME }}
+          tenant1-apitoken: ${{ secrets.TENANT1_APITOKEN }}
+          tenant1-oteltoken: ${{ secrets.TENANT1_OTELTOKEN }}
+          tenant1-oauth-client-id: ${{ secrets.TENANT1_OAUTH_CLIENT_ID }}
+          tenant1-oauth-secret: ${{ secrets.TENANT1_OAUTH_SECRET }}
+          tenant2-name: ${{ secrets.TENANT2_NAME }}
+          tenant2-apitoken: ${{ secrets.TENANT2_APITOKEN }}
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+
+  run-ocp-matrix:
+    name: Run in OpenShift
+    environment: E2E
+    strategy:
+      matrix:
+        flc_env: [ dto-ocp-4-10, dto-ocp-4-11, dto-ocp-4-12, dto-ocp-4-13 ]
```

Comment:
4.10 is out of support since 1st April -> https://docs.dynatrace.com/docs/setup-and-configuration/technology-support/support-model-for-kubernetes

Reply:
@luhi-DT am I reading table incorrectly?
>Mar 1, 2025
>End of support (OpenShift)

<img width="844" alt="Screenshot 2024-06-03 at 09 13 45" src="https://github.com/Dynatrace/dynatrace-operator/assets/1374633/fdb2c6ca-854c-460f-a083-ebf00b76650d">

Reply:
good point, lets leave it for now, but i am also confused about this